src/WebJobs.Script.WebHost/Management/WebFunctionsManager.cs

```diff
@@ -144,12 +151,22 @@ await functionMetadata
                 await FileUtility.WriteAsync(dataFilePath, functionMetadata.TestData);
             }
 
+            var metadata = (await _hostFunctionMetadataProvider.GetFunctionMetadataAsync(_languageWorkerOptions.CurrentValue.WorkerConfigs, configChanged))
+                .FirstOrDefault(metadata => Utility.FunctionNamesMatch(metadata.Name, name));
+
+            var success = false;
+            FunctionMetadataResponse functionMetadataResult = null;
+            if (functionMetadata != null)
```

Comment:
Can functionMetadata be null here? We are checking properties of it in code above this, so it would be odd for it to be null at this point? Would it be better to add a null check on `functionMetadata` at the start of this method?

Reply:
This was a bug, i was supposed to check the updated function metadata here
https://github.com/Azure/azure-functions-host/pull/10678/files#diff-203384510505c08751e66abe5167ac5534fe369ed79ffa4d424d9a4eb8847fe8R154

Reply:
I have fixed it.